### ISSUE TITLE
Rotating has surprising dimensions

### ISSUE THREAD

**Issue by ben-manes** [coobird/thumbnailator#99]
When I rotate an image by 90 degrees the result is squished.

Take for example this [image](https://d1740fs3ymhq5p.cloudfront.net/43f1143e-966c-42c7-a4e6-312b57bd9944/editor/5f094ba9-fb1d-480b-87e7-44eb56ea758a_s100.jpg) (1238 × 1650) and the transformation,

``` java
Thumbnails.of(input.toFile()).scale(1.0).rotate(angle).toFile(output.toFile());
```

The [output image](https://d1740fs3ymhq5p.cloudfront.net/43f1143e-966c-42c7-a4e6-312b57bd9944/editor/acb61198-31c2-4d69-abc6-e27031323eb6.jpg) has the dimensions 1238 × 929. I had expected it to be 1650 x 1238.

Since the rotation can apply any angle, I'm not sure if this is a faulty assumption or a mistake in the library.

**Comment by ben-manes** [comment]
Surprisingly this works as expected if I use `Rotation` directly.

``` java
BufferedImage in = ImageIO.read(input.toFile());
BufferedImage out = Rotation.newRotator(angle).apply(in);
Thumbnails.of(out).scale(1.0).toFile(output.toFile());
```

**Comment by coobird** [comment]
@ben-manes, thank you for filing this issue and sorry for not getting back to you in a timely manner.

I have tried running the code you provided along with the image you provided and found that I wasn't able to reproduce your issue.

Could you give me more information regarding which version of Thumbnailator, JDK (including vendor), and OS you're using?

I tried a couple times in the past to reproduce your issue (including writing additional tests that may reproduce the issue), but have not been able to.

**Comment by ben-manes** [comment]
Thanks @coobird. I can't seem to reproduce it either, so that's good. I do recall it failing, but clearly not anymore.

At the time I was running into a lot of problems with bad metadata in the jpegs produced by Android cameras. This caused BufferedImage to provide incorrect dimensions, so I switched all my processing to OpenCV (via javacv). Perhaps I attributed the issue to the wrong place, at the time, though I don't know why my claimed workaround would be any different.
